Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[workspace]
+
+members = [
+    "controller",
+    "agent",
+    "networking",
```

Comment:
Can you rename `networking` to `network` please. It's to prevent long name in the future

Reply:
done !